pandas/tests/io/json/test_pandas.py

```diff
@@ -1262,3 +1262,12 @@ def test_index_false_error_to_json(self, orient):
                "'orient' is 'split' or 'table'")
         with pytest.raises(ValueError, match=msg):
             df.to_json(orient=orient, index=False)
+
+    @pytest.mark.parametrize('orient', ['split', 'table'])
+    def test_index_false_from_json_to_json(self, orient):
+        # GH25170
+        # Test index=False in from_json to_json
+        df = DataFrame({'a': [1, 2], 'b': [3, 4]})
+        dfjson = df.to_json(orient=orient, index=False)
+        result = read_json(dfjson, orient=orient)
+        assert_frame_equal(result, df)
```

Comment:
Usually like to do `assert_frame_equal(result, expected)` so if you could update naming conventions here would be great